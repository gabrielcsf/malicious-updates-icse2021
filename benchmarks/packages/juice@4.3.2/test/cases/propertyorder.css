.b {
  padding: 20px;
  padding-top: 10px;
  color: red;
}
.a {
  padding-top: 0;
  padding: 5px;
  padding: 10px;
}
