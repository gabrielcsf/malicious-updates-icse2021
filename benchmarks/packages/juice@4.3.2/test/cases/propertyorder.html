<div class="a b"></div>
