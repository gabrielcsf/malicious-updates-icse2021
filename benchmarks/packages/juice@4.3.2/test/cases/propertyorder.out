<div class="a b" style="color: red; padding-top: 0; padding: 5px; padding: 10px;"></div>
